plots/plot.py

```diff
@@ -98,26 +98,24 @@ def printHelp():
     exit(0)
 
 def printSummaryStats(dataset):
-    print("Model population performance:\n{0:^30} {1:^5} {2:^5} {3:^5}".format("Decision Model", "Extinct", "Worse", "Better"))
+    print(f"Model population performance:\n{"Decision Model":^30} {"Extinct":^5} {"Worse":^5} {"Better":^5}")
     for model in dataset:
         better = dataset[model]["runs"] - (dataset[model]["died"] + dataset[model]["worse"])
-        print("{0:^30}: {1:^5} {2:^5} {3:^5}".format(model, dataset[model]["died"], dataset[model]["worse"], better))
+        print(f"{model:^30}: {dataset[model]["died"]:^5} {dataset[model]["worse"]:^5} {better:^5}")
 
 def printRawData(dataset, totalTimesteps, outfile):
     file = open(outfile, 'w')
     columnHeads = "timestep"
     for model in dataset:
         for metric in ["pop", "mttl", "strv", "comd", "welt", "maad"]:
-            columnHeads += " {0}_{1}".format(model, metric)
+            columnHeads += f" {model}_{metric}"
     columnHeads += '\n'
     file.write(columnHeads)
 
     for i in range(totalTimesteps + 1):
         line = str(i)
         for model in dataset:
-            line += " {0} {1} {2} {3} {4} {5}".format(dataset[model]["meanMetrics"]["population"][i], dataset[model]["meanMetrics"]["agentMeanTimeToLive"][i],
-                                                  dataset[model]["meanMetrics"]["agentStarvationDeaths"][i], dataset[model]["meanMetrics"]["agentCombatDeaths"][i],
-                                                  dataset[model]["meanMetrics"]["agentWealthTotal"][i], dataset[model]["meanMetrics"]["meanAgeAtDeath"][i])
+            line += f" {dataset[model]["meanMetrics"]["population"][i]} {dataset[model]["meanMetrics"]["agentMeanTimeToLive"][i]} {dataset[model]["meanMetrics"]["agentStarvationDeaths"][i]} {dataset[model]["meanMetrics"]["agentCombatDeaths"][i]} {dataset[model]["meanMetrics"]["agentWealthTotal"][i]} {dataset[model]["meanMetrics"]["meanAgeAtDeath"][i]}"
```

Comment:
I think we should still split this line up for readability, maybe something like this:
```python
line += f" {dataset[model]["meanMetrics"]["population"][i]} {dataset[model]["meanMetrics"]["agentMeanTimeToLive"][i]}"
line += f" {dataset[model]["meanMetrics"]["agentStarvationDeaths"][i]} {dataset[model]["meanMetrics"]["agentCombatDeaths"][i]}"
line += f" {dataset[model]["meanMetrics"]["agentWealthTotal"][i]} {dataset[model]["meanMetrics"]["meanAgeAtDeath"][i]}"

```